src/main/java/com/ericsson/eiffel/remrem/generate/helper/RemremJarHelper.java

```diff
@@ -0,0 +1,101 @@
+package com.ericsson.eiffel.remrem.generate.helper;
+
+import java.io.File;
+import java.io.IOException;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.net.URLClassLoader;
+
+import javax.annotation.PostConstruct;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import com.ericsson.eiffel.remrem.generate.cli.CLIOptions;
+import com.ericsson.eiffel.remrem.generate.listener.DirectoryWatchService;
+import com.ericsson.eiffel.remrem.generate.listener.SimpleJarDirectoryWatchService;
+
+@Component("jarHelper") public class RemremJarHelper {
+
+    @Value("${jar.path}") private String jarPath;
+
+    public String getJarPath() {
+        return jarPath;
+    }
+
+    public void setJarPath(String jarPath) {
+        this.jarPath = jarPath;
+    }
+	
+    @PostConstruct public void init() {
+        try{
+            if(CLIOptions.getCommandLine()==null ){
+                addJarsToClassPath(jarPath);
+                lookupForJarFileChanges();
+            }
+            else if(!CLIOptions.getCommandLine().hasOption("jp")){
+                addJarsToClassPath(jarPath);
+            }
+        }catch(IOException e){
+            e.printStackTrace();
+        } catch (Exception e) {
```

Comment:
There is no method throwing anything but IOException so we do not need to catch Exception.